lib/api/wpc_proto.h

```diff
@@ -29,12 +29,29 @@ typedef enum
     APP_RES_PROTO_NOT_ENOUGH_MEMORY,
 } app_proto_res_e;
 
-
-app_proto_res_e WPC_Proto_initialize(const char * port_name,
-                                     unsigned long bitrate,
-                                     char * gateway_id,
+/**
+ * \brief        Init protobuf interface
+ * \param[in]    gateway_id
+ *               Pointer to gateway id string
+ * \param[in]    gateway_model
+ *               Pointer to gateway model string, "" if not available
+ * \param[in]    gateway_version
+ *               Pointer to gateway version string, "" if not available
+ * \param[in]    sink_id
+ *               Pointer to the sink id string
+ * \return       Return code of the operation
+ */
+app_proto_res_e WPC_Proto_initialize(char * gateway_id,
```

Comment:
For me, port_name and bitrate must remain in API.
WPC_Proto is wrapper on top of WPC_ so it looks a bit strange if you have to explicitly call WPC_initialize before  WPC_Proto_initialize

Reply:
Done, will push after test